src/main/java/mate/academy/springboot/swagger/util/SortUtil.java

```diff
@@ -0,0 +1,28 @@
+package mate.academy.springboot.swagger.util;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.data.domain.Sort;
+import org.springframework.stereotype.Component;
+
+@Component
+public class SortUtil {
```

Comment:
util class must have a static method, remove `Component` annotation and change `sort` method